source/index.html.md.erb

```diff
@@ -22,10 +22,10 @@ New to the OPG team? [Read how to get started here](documentation/get_started)
 - [Terraform Practices](documentation/guides/terraform_practices)
 - [CircleCI Dockerhub Authentication](documentation/guides/circleci_dockerhub)
 - [Terraform Module](documentation/guides/terraform_modules)
+- [Repository README Principles](documentation/guides/repo_principles)
 
 ## Communites of practice
 
-- [Accessibility Community of Practice](documentation/cops/accessability)
```

Comment:
Is this now longer active?

Reply:
We had one meeting and it morphed into the FE COP.